src/net.cpp

```diff
@@ -2360,6 +2404,11 @@ bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
         semAddnode = std::make_unique<CSemaphore>(nMaxAddnode);
     }
 
+    for (int n = 0; n < NET_MAX; n++) {
+        enum Network net = (enum Network)n;
+        m_network_conn_counts[net] = 0;
+    }
+
```

Comment:
This is unnecessary, you can just drop it. The first time `operator[]` is used for a given key a default-initialized value is inserted for that key, which in our case is `0`.

Reply:
changed back to array and this was not added back but it is not needed because the array is default-initialized: `std::array<...> foo = {};`